state/src/impls/top_level.rs

```diff
@@ -465,6 +533,11 @@ impl TopLevelState {
                 return Err(ParcelError::InconsistentShardOutcomes.into())
             }
         }
+
+        let unwrapped_amount = transaction.unwrapped_amount();
+        if !unwrapped_amount.is_zero() {
```

Comment:
Can unwrapped_amount be zero?

Reply:
On other transaction types can be zero.

Reply:
Currently, only on `AssetUnwrapCCC` type will return non-zero value. I don't think it should be a function.